src/main/java/com/revature/models/Product.java

```diff
@@ -38,18 +38,30 @@ public class Product {
     @OneToMany(orphanRemoval = true)
     @JoinColumn(name = "product_id")
 
-    @JsonBackReference
+
     @ToString.Exclude
     private Set<Review> reviews = new LinkedHashSet<>();
 
     @OneToMany(orphanRemoval = true)
     @JoinColumn(name = "product_id")
-    @JsonBackReference
```

Comment:
I'm not sure what this is, so can you explain why you removed that annotation?

Reply:
That annotation will cause  the error("Content type 'application/json;charset=UTF=8' not support") to any API endpoint related to the Product.java.

The purpose of @JsonBackReference is to prevent StackOverflow errors. But @JsonIgnoreProperties also fixes the StackOverflow error, we don't need JsonBackReference anymore. I did confirm with Sam about the changes when I found the error, but it was like 9 pm est time.